pkg/descheduler/strategies/duplicates.go

```diff
@@ -167,6 +168,9 @@ func RemoveDuplicatePods(
 								duplicatePods[ownerKey] = make(map[string][]*v1.Pod)
 							}
 							duplicatePods[ownerKey][node.Name] = append(duplicatePods[ownerKey][node.Name], pod)
+							// We would like to see the same copy
+							metrics.DuplicatePods.With(map[string]string{"workloadName": ownerKey.name,
```

Comment:
does this need to reset or account for when duplicate pods are removed? Or is it just infinitely increasing?

Reply:
What's the workload in this context? Might still be more user friendly to use owner/kind directly. E.g. in case of a deployment, one gets RS.

Reply:
@damemi - You're right. I have used a CounterVec and couldn't find decrement method associated with it. I changed it to Guage now and we should be good.

Reply:
I am not in a favor of prefixing the names with `workload` keyword and inventing new names where we can still use the existing `kind` and `name`. Unless there's clear benefit to consumers of the new metric.

Reply:
Kind and Name are fine but we want to be generic enough to be used in workload context. The workload can be anything STS, deployment etc. I don't want to harp on this. I'll change it.